### DISCUSSION TITLE
Problems about the scale of label in graph-level regression problem

### DISCUSSION THREAD

**Discussion by YiKangOY** [pyg-team/pytorch_geometric#7148] [Q&A] [answered]
Hi all!

Recently I had a graph-level regression problem. The number of nodes varies from 450 to 140000. I used GIN model with a sum pooling to get the embedding of the graphs. Then I use MLP to regress the labels. The labels vary from 30 to 20000. 

I would like to ask if there is any need to scale the data to make the labels lie in a narrower range?

**Comment by EdisonLeeeee** [comment, marked as answer]
In most cases, you might need a log transform to the labels to normalize the distribution and improve the performance of the regression model.

**Reply by YiKangOY** [reply]
Thanks, I will give it a try.
